### PR TITLE
[p5.js hi]: Constants, Foundation and Transform References

### PR DESCRIPTION
**Pull request by SanjaySinghRajpoot**
Foundation - 11 
Constants - 7 
Transform - 10

### CODE REVIEW

File: src/content/reference/hi/p5/for.mdx [hunk 1]
```diff
@@ -1,186 +1,109 @@
...
-  <code>100</code>.</li>
+   <ul>
 
-  <li><code>x += 20</code>  tells the loop to count up by <code>20</code> at the
-  end of each iteration.</li>
+   <li><code>let x = 10</code> लूप को गिनती शुरू करने के लिए कहता है
+   <code>10</code> और वेरिएबल का उपयोग करके पुनरावृत्तियों पर नज़र रखें
+   <code>x</code>.</li>
 
-  </ul>
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/for.mdx:45]: this change is unnecessary imo

File: src/content/reference/hi/p5/rotate.mdx [hunk 2]
```diff
@@ -1,65 +1,18 @@
 ---
 title: rotate
-module: Transform
-submodule: Transform
+module: परिवर्तन
+submodule: परिवर्तन
 file: src/core/transform.js
 description: >
-  <p>Rotates the coordinate system.</p>
+  <p>समन्वय प्रणाली को घुमाता है।</p>
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/rotate.mdx:7]: as discussed use 'निर्देशांक' for coordinate

**Review by Divyansh013 — CHANGES_REQUESTED** [review]

File: src/content/reference/hi/p5/for.mdx (hunk 1 above)
Comment (SanjaySinghRajpoot) [reply src/content/reference/hi/p5/for.mdx:45]: can you please elaborate more on this?

**Comment by Divyansh013** [comment]
@TanviKumar PTAL

**Review by TanviKumar — APPROVED** [review]

**Comment by Divyansh013** [comment]
@Qianqianye @limzykenneth we can merge this PR

**Comment by Qianqianye** [comment]
Hi @SanjaySinghRajpoot @Divyansh013 @TanviKumar, could you fix this PR so it could pass the checks? Thanks!

**Comment by TanviKumar** [comment]
@Qianqianye you can merge this now

**Comment by Qianqianye** [comment]
Looks great, thank you all!

**Pull request merged**